scripts/release.sh

```diff
@@ -485,7 +485,6 @@ function main() {
   parse_flags $@
   # Log what will be done and where.
   banner "Release configuration"
-  echo "- gcloud user: $(gcloud config get-value core/account)"
```

Comment:
Please, don't remove it, as this is useful for debugging. I'm totally OK with dealing with gcloud not being installed though.

Reply:
What's the use of ibmcloud user here? For gcloud, it's a useful debugging piece because of the Prow job user. AFAICU ibmcloud isn't used anywhere.

Reply:
I have no idea how gcloud is used in these so I just was guessing at what's going on.  Is the prow user not listed any place else in the output? Like in the prow debug stuff?

Reply:
gcloud is used when publishing releases. Prow has no debug stuff for jobs, it's up to the job doing that.